map-of-the-web/motw/motw/urls.py

```diff
@@ -2,16 +2,6 @@
 
 The `urlpatterns` list routes URLs to views. For more information please see:
     https://docs.djangoproject.com/en/2.0/topics/http/urls/
-Examples:
```

Comment:
This commit should be "squashed" into the commit that originally added this.  That way these lines will never be in the repository history and the commit log will be a bit cleaner.

you can do this by typing:

```
git rebase -i 0790ea2251818166db029032718e8200f4b9c292
```

(Note: `0790ea2251818166db029032718e8200f4b9c292` is the hash of the first commit in this branch, which means rebase will let you edit and merge all of the commits in your work so far.

The interface that shows up lets you reorder commits, and combine commits into one another.  Please let me know if you have any questions on how to use it!